packages/deployment/package.json

```
@@ -22,6 +22,7 @@
     "@agoric/assert": "^0.6.0",
     "@endo/init": "^0.5.56",
     "@endo/marshal": "^0.8.5",
+    "better-sqlite3": "^8.4.0",
```

We've already got 2 versions of better-sqlite3; let's not add a 3rd.

```suggestion
    "better-sqlite3": "^8.2.0",
```

```
$ grep better-sqlite3 */package.json
deployment/package.json:    "better-sqlite3": "^8.4.0",
SwingSet/package.json:    "@types/better-sqlite3": "^7.5.0",
SwingSet/package.json:    "better-sqlite3": "^8.2.0",
swing-store/package.json:    "better-sqlite3": "^8.2.0"
swing-store/package.json:    "@types/better-sqlite3": "^7.5.0",
telemetry/package.json:    "better-sqlite3": "^8.2.0",
```

---

switched to 8.2.0.

---

we just have one. the 7.5.0 is typedefs. they haven't been published for 8.x yet https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/better-sqlite3/index.d.ts

but yeah we should stick to 8.2 consistently here and upgrade them all separately